Python/codegen.c

```diff
@@ -823,7 +823,14 @@ codegen_process_deferred_annotations(compiler *c, location loc)
     PySTEntryObject *ste = SYMTABLE_ENTRY(c);
     assert(ste->ste_annotation_block != NULL);
     void *key = (void *)((uintptr_t)ste->ste_id + 1);
-    if (codegen_setup_annotations_scope(c, loc, key,
+
+    // Get the first annotation location
```

Comment:
assert here that deferred_anno is not empty.

Reply:
I didn't add it because we already checked it above. https://github.com/python/cpython/blob/main/Python/codegen.c#L804

Reply:
That checks for NULL. But before you access an element you want to know it's not empty.

Reply:
Done